Source/Core/VideoCommon/VertexShaderManager.cpp

```diff
@@ -683,6 +684,14 @@ void VertexShaderManager::SetViewportChanged()
 void VertexShaderManager::SetProjectionChanged()
 {
 	bProjectionChanged = true;
+
+	if (g_renderer->GetPostProcessor())
```

Comment:
This should be its own function that takes the projection type. In particular, the postprocessor should have a method called ProjectionChanged or something that takes the projection type, which keeps this code local to the postprocessor rather than putting it out into the vertex shader manager.